python/test/unit/language/test_core.py

```diff
@@ -2988,7 +2994,7 @@ def convert_fp8_to_fp32(x, device, dtype_str):
 @pytest.mark.parametrize(
     "M, N, K, num_warps, col_a, col_b, epilogue, input_precision, in_dtype, out_dtype, kpack",
     [(*shape, 4, False, False, epilogue, input_precision, in_dtype, out_dtype, 1)
-     for shape in [(64, 64, 64), (32, 32, 32), (16, 16, 16)]
+     for shape in [(64, 64, 64), (32, 32, 32), (16, 16, 16), (16, 16, 8)]
```

Comment:
It's better to add this new (16, 16, 8) only is_hip() is true, rather than adding it now and skip it later.